packages/core/src/components/BaseInput/BaseInput.styles.tsx

```diff
@@ -122,14 +121,14 @@ export const { staticClasses, useClasses } = createClasses("HvBaseInput", {
   inputRootInvalid: { borderColor: theme.colors.negative },
   inputRootReadOnly: {
     borderColor: theme.colors.secondary_60,
-    backgroundColor: "transparent",
+    backgroundColor: theme.colors.atmo3,
```

Comment:
This looks like `atmo2` [in the spec](https://designsystem.hitachivantara.com/6c705d900/p/6700f7-input/b/50959e) 🤔

Reply:
I'll revert back the background color. It will most likely be `atmo3` in the future but DS is still making some tests. Once it's official we'll change it.